### PR TITLE
Add seattle file to class repo

### LINKED ISSUE #33: rajkorde tracks progress on GitHub Flow
- [x] create a **branch** (git)
- [ ] add some commits (git)
- [ ] open a _pull request_ (GitHub)
- [ ] additional commits (git)
- [ ] merge (git)

### PR DESCRIPTION
**Pull request by rajkorde**
This PR adds a seattle file to the class request. This PR closes #33 

- @iservan hello you ever been to seattle?
- whats your favorite holiday location?

### CODE REVIEW

File: rajkorde-seattle.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Seattle, WA
+
+### Restaurants
+- Staple and Fancy
+
+### Attractions for Tourists
+- Space Needle
+- EMP
```
Comment (githubteacher) [inline rajkorde-seattle.md]: @rajkorde what's EMP?

**Comment by githubteacher** [comment]
> whats your favorite holiday location?

I live in FL so it's always HOT, somewhere cool! 🍨

File: rajkorde-seattle.md (hunk 1 above)
Comment (rajkorde) [inline rajkorde-seattle.md]: Experience Music Project. its a museum.

Comment (rajkorde) [inline rajkorde-seattle.md]: clarified what EMP means

**Comment by rajkorde** [comment]
I'll merge when I have at least 1 :sheep:

**Comment by neonblueflame** [comment]
:sheep: :+1:

**Pull request merged**